src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -310,16 +319,19 @@ export const Participants = () => {
         content: () => stringSecretIds,
         value: stringSecretIds,
         width: ParticipantsColumnsWidth.Default,
+        onClick: defaultOnClick,
       },
       nicknames: {
         content: () => stringNicknames,
         value: stringNicknames,
         width: ParticipantsColumnsWidth.Default,
+        onClick: defaultOnClick,
       },
       tags: {
         content: () => <>--</>,
```

Comment:
Also while you're here, I'd previously created a shared component specifically for the case to correctly style the `--` placeholder when the value is falsy (it's supposed to be a lighter grey):

```suggestion
        // TODO: Replace `null` with tags component when available
        content: () => <StyledMaybeEmpty>{null}</StyledMaybeEmpty>,
```